pkg/cmd/root.go

```diff
@@ -21,6 +21,7 @@ var rootCmd = &cobra.Command{
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&helpers.LogLevel, "log-level", "l", "info", helpers.LogLevelMsg)
 	rootCmd.PersistentFlags().BoolVar(&helpers.ColoredOutput, "color", false, helpers.ColoredOutputMsg)
+	rootCmd.PersistentFlags().StringVarP(&helpers.KubeConfigPath, "kubeconfig", "", "", "kube config file Path.")
```

Comment:
This is not good. This means `kubeconfig` flag is available to ALL commands. While this is desirable in the future, this is not supported by all commands yet. This is going to cause a lot of confusions.

Either:
- Move this to the get cluster command only. OR 
- Remove this flag entirely and do this in a separate PR.